dss/api/bundles/__init__.py

```diff
@@ -83,12 +83,20 @@ def get(
             file_version['url'] = str(UrlBuilder().set(
                 scheme=_replica.storage_schema,
                 netloc=_replica.checkout_bucket,
-                path=compose_blob_key(file),
+                path="{}/{}".format(
+                    get_dst_bundle_prefix(uuid, version),
```

Comment:
This will fail when version is None.

Reply:
Fixed.  Added tests to cover this scenario.